hydra-node/src/Hydra/Chain/Direct/Wallet.hs

```diff
@@ -138,9 +137,7 @@ data TinyWallet m = TinyWallet
   -- wallet is still initializing.
   }
 
-data SomePParams
-  = BabbagePParams (PParams Babbage)
-  | ConwayPParams (PParams Conway)
+data SomePParams = ConwayPParams (PParams Conway)
```

Comment:
Note: We _could_ delete this entirely; but maybe it's convenient to leave it in anticipation of supporting another Era at some point?

Reply:
Maybe we could document the type explaining that